src/function.js

```diff
@@ -16,20 +16,25 @@ let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
   }
 
   function returnNewArrar (event) {
-    console.log(event);
       const target1 = event.target.id;
-      console.log(target1);
       const targetIdNumber =parseInt(target1.match(/\d+/)[0], 10);
-      console.log(JSON.stringify(targetIdNumber));
       const taskIndex = tasks.findIndex(task => task.index === targetIdNumber + 1);
-      console.log(taskIndex);
-      tasks[taskIndex].completed = true;
-      const nextElemSinling1 = event.target.nextElementSibling;
-      console.log(nextElemSinling1);
-      nextElemSinling1.style.textDecoration = 'line-through' ;
+      status(tasks[taskIndex])
+      classToggle(event)
       localStorage.setItem('tasks', JSON.stringify(tasks));
   }
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if your app should not reload while clearing the completed task 🤔 because right now your app reloads after clearing the completed todos 😨 which is not expected behavior 🤔 It would be nice if you follow the project requirements and your app won't reload while doing the crud operations 👍

**Hint:** Kindly make a display method to display todos after doing crud operations 👍 What this method will do ⬇️
1. First it will clear all take the localStorage and filter the completed items
2. Second, it will remove all the todos from your DOM
3. Third it will render a new list of todos from local storage